assets/base.css

```diff
@@ -2231,6 +2231,23 @@ details[open] > .header__icon--menu .icon-hamburger {
     transform var(--duration-default) ease;
 }
 
+.header__submenu.list-menu--disclosure {
+  --shadow-horizontal-offset: var(--popup-drawer-shadow-horizontal-offset);
+  --shadow-vertical-offset: var(--popup-drawer-shadow-vertical-offset);
+  --shadow-blur-radius: var(--popup-drawer-shadow-blur-radius);
+  --shadow-opacity: var(--popup-drawer-shadow-opacity);
```

Comment:
We'd have to have these variables under a class or `root`, but not sure if that would make sense or be readable since its just for these elements 🤔 

@tyleralsbury What do you think?

Reply:
I think that would reduce a lot of the repetition. And then if you want to overwrite on a section level then, only then you add some section specific variable overwrite 🤔

Reply:
Just not sure if root would end up bloated. If we do that we should do for all of the different settings

Reply:
The way I did it was by using the variable assigned by the setting directly:

**In the multicolumn CSS:**

``` css
.multicolumn-card {
  border: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
  padding: var(--text-boxes-padding);
  border-radius: var(--text-boxes-radius);
  box-shadow: var(--text-boxes-shadow-horizontal-offset)
    var(--text-boxes-shadow-vertical-offset)
    var(--text-boxes-shadow-blur-radius)
    rgba(var(--color-foreground), var(--text-boxes-shadow-opacity));
}
```

**In `theme.liquid`:**

``` liquid
        --text-boxes-shadow-opacity: {{ settings.text_boxes_shadow_opacity | divided_by: 100.0 }};
        --text-boxes-shadow-horizontal-offset: {{ settings.text_boxes_shadow_horizontal_offset }}px;
        --text-boxes-shadow-vertical-offset: {{ settings.text_boxes_shadow_vertical_offset }}px;
        --text-boxes-shadow-blur-radius: {{ settings.text_boxes_shadow_blur_radius }}px;
```

It does help to reduce repetition, so I think it overall is a bit cleaner.